runtime/vm/FFITypeHelpers.cpp

```diff
@@ -18,156 +18,4 @@
  * [2] https://openjdk.org/legal/assembly-exception.html
  *
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
- *******************************************************************************/
```

Comment:
This file is not used. I think it and `FFITypeHelpers.hpp` should simply be deleted.